PWGLF/Tasks/LFNucleiBATask.cxx

```diff
@@ -74,8 +74,8 @@ struct LFNucleiBATask {
 
     // trackQA
     histos.add<TH1>("tracks/h1Eta", "pseudoRapidity; #eta; counts", HistType::kTH1F, {{200, -1.0, 1.0}});
-    histos.add<TH1>("tracks/h1VarPhi", "#phi; #phi; counts", HistType::kTH1F, {{80, -0.5, 7.5}});
-    histos.add<TH2>("tracks/h2EtaVsPhi", "#eta (TOF) vs #phi; #eta; #phi", HistType::kTH2F, {{200, -1.0, 1.0}, {80, -0.5, 7.5}});
+    histos.add<TH1>("tracks/h1VarPhi", "#phi; #phi; counts", HistType::kTH1F, {{63, 0.0, 6.3}});
```

Comment:
So this is the only change right?

Reply:
Yes, I just changed that line, but missing one bracket make everything "explode".